mmseg/models/losses/tversky_loss.py

```diff
@@ -87,15 +93,18 @@ def __init__(self,
                  ignore_index=255,
                  alpha=0.3,
                  beta=0.7,
+                 gamma=1.0,
                  loss_name='loss_tversky'):
         super(TverskyLoss, self).__init__()
         self.smooth = smooth
         self.class_weight = get_class_weight(class_weight)
         self.loss_weight = loss_weight
         self.ignore_index = ignore_index
         assert (alpha + beta == 1.0), 'Sum of alpha and beta but be 1.0!'
+        assert gamma >= 1.0, 'gamma should be at least 1.0!'
```

Comment:
Since there is an assertion statement, we might add a unit test for it.

Reply:
I presume I don't need to do anything for this comment?